tests/d2s_test.rs

```diff
@@ -51,13 +51,15 @@ fn test_ryu() {
 
 #[test]
 fn test_random() {
+    let n = if cfg!(miri) { 100 } else { 1000000 };
     let mut buffer = ryu::Buffer::new();
-    for _ in 0..1000000 {
+    for _ in 0..n {
         let f: f64 = rand::random();
         assert_eq!(f, buffer.format_finite(f).parse().unwrap());
     }
 }
 
+#[cfg(not(miri))]
```

Comment:
FWIW, the usual alternative I use these days is `#[cfg_attr(miri, ignore)]`. Then at least it is clear in the output that we are skipping some tests.

It doesn't really make a difference, though.